tensorflow_serving/apis/BUILD

```diff
@@ -156,6 +156,31 @@ serving_go_grpc_library(
     deps = [":prediction_service_go_proto"],
 )
 
+serving_proto_library(
+    name = "model_management_proto",
+    srcs = ["model_management.proto"],
+    cc_api_version = 2,
+    go_api_version = 2,
+    java_api_version = 2,
+    deps = [
+        "//tensorflow_serving/config:model_server_config_proto",
+        "//tensorflow_serving/util:status_proto",
+    ],
+)
+
+serving_proto_library(
+    name = "model_management_service_proto",
```

Comment:
Actually, rather than defining a new service, let's add this new RPC method to the existing ModelService service (defined in model_service.proto). The comment in that service would have to be updated to become broader (not just obtaining information, but also updating the set of models or other config).